### PR TITLE
impl get_peer()

### PR DESCRIPTION
**Pull request by jmcook1186**
implements `get_peer()` endpoint

### CODE REVIEW

File: src/api_client.rs [hunk 1]
```diff
@@ -459,8 +459,13 @@ impl Client {
         }
     }
 
-    pub async fn get_peer(peer_id: Multiaddr) -> Result<PeerDescription, Error> {
-        unimplemented!("")
+    pub async fn get_peer(&self, peer_id: PeerId) -> Result<PeerDescription, Error> {
+        let id = serde_json::to_string(&peer_id)
+            .unwrap()
+            .replace(&['\"'][..], "");
```
Comment (ralexstokes) [inline src/api_client.rs]: I'd rather just impl `fmt::Display` in the upstream -- can you make a PR to do that and then refactor this?

Comment (ralexstokes) [reply src/api_client.rs]: can add a `fmt::Display` impl here: https://github.com/ralexstokes/ethereum-consensus/blob/main/src/networking.rs#L24

and likely re-use the `b58` code from the `serde` module there

**Review by ralexstokes — COMMENTED** [review]
excellent! I think should just upstream the `to_string` logic for `PeerId` but the rest looks great!

File: src/api_client.rs (hunk 1 above)
Comment (jmcook1186) [reply src/api_client.rs]: ok - [this PR to eth_consensus ](https://github.com/ralexstokes/ethereum-consensus/pull/165) upstreams the display logic, and the API call is simplified.

File: src/api_client.rs [hunk 2]
```diff
@@ -459,8 +459,11 @@ impl Client {
         }
     }
 
-    pub async fn get_peer(peer_id: Multiaddr) -> Result<PeerDescription, Error> {
-        unimplemented!("")
+    pub async fn get_peer(&self, peer_id: PeerId) -> Result<PeerDescription, Error> {
+        let id = &peer_id;
+        let result: Value<PeerDescription> =
+            self.get(&format!("/eth/v1/node/peers/{}", id)).await?;
```
Comment (ralexstokes) [inline src/api_client.rs]:
```suggestion
        let result: Value<PeerDescription> =
            self.get(&format!("/eth/v1/node/peers/{}", peer_id)).await?;
```

**Review by ralexstokes — COMMENTED** [review]
we can skip the intermediate binding

**Review by ralexstokes — APPROVED** [review]
fantastic!

**Pull request merged**